namer/serversets/src/main/scala/com/twitter/finagle/serversets2/BouyantZkResolver.scala

```diff
@@ -0,0 +1,15 @@
+package com.twitter.finagle.serverset2
+
+import com.twitter.finagle.{Addr, Resolver}
+import com.twitter.util.Var
+
+class BouyantZkResolver(zk2: Zk2Resolver) {
```

Comment:
What is the purpose of this class?  Can't we just pass a Zk2Resolver to the ServersetNamer directly?

Reply:
I needed access to `private[twitter] Zk2Resolver.addrOf` [to resolve the shard component](https://github.com/twitter/finagle/blob/198d0eb3037b9fc7bfbd671b4aa3c1fd8ab1580a/finagle-serversets/src/main/scala/com/twitter/finagle/serverset2/Zk2Resolver.scala#L264) so I declared `BouyantZkResolver` in the com.twitter.finagle.serverset2 package for access.